website/blog/2024-05-14-Hyperview-Navigation.md

```diff
@@ -0,0 +1,17 @@
+---
+author: Hardin Gray
+authorURL: https://github.com/hgray-instawork
+title: Hyperview Navigation Announcement
+---
+
+## Hyperview navigation
```

Comment:
When I look at the preview, this header is not needed, the title gets turned into an H1